internal/storage/storage_handle.go

```diff
@@ -152,7 +197,7 @@ func NewStorageHandle(ctx context.Context, clientConfig storageutil.StorageClien
 		storage.WithPolicy(storage.RetryAlways),
 		storage.WithErrorFunc(storageutil.ShouldRetry))
 
```

Comment:
The Go SDK code appears to lack retries for creating and deleting folders, while retries for getting and renaming folders are infinite. We cannot configure retries for the control client from our side.

https://github.com/googleapis/google-cloud-go/blob/main/storage/control/apiv2/storage_control_client.go#L73
@tritone can you please confirm?

Reply:
@Tulsishah is correct about the defaults, but you can also configure retries per-call with the GAPIC client. Use https://pkg.go.dev/github.com/googleapis/gax-go/v2#WithRetry to configure; this can be passed to any GAPIC generated RPC.